microsoft-edge/webview2/release-notes.md

```diff
@@ -1,1194 +1,1251 @@
-*  Turned on requesting `window.open()` in script.
-
-
-<!-- ====================================================================== -->
-## 0.8.149
-
-[NuGet package for WebView2 SDK 0.8.149](https://www.nuget.org/packages/Microsoft.Web.WebView2/0.8.149)
-
-This version of the WebView2 SDK requires Microsoft Edge version 76.0.149.0 or higher.
-
-Initial developer preview release.
-
-
-<!-- ====================================================================== -->
-## See also
-
-*  [Contacting the Microsoft Edge WebView2 team](contact.md)
+---
+title: Release Notes for the WebView2 SDK
+description: Release notes for Microsoft Edge WebView2 for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 11/29/2021
+---
+# Release Notes for the WebView2 SDK
+
+The WebView2 team updates the [WebView2 SDK](https://www.nuget.org/packages/Microsoft.Web.WebView2) on a four-week cadence.  This article contains the latest information on product announcements, additions, modifications, and breaking changes to the APIs.
+
+WebView2 bug fixes, such as the fixes listed below, are either Runtime-specific or SDK-specific.
+
+
+<!-- ====================================================================== -->
+## Recommended browser channel and Runtime
+
+Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet package.  The WebView2 team recommends the following:
+
+*  Use the Canary preview channel of Microsoft Edge when you develop using a prerelease version of the WebView2 SDK package.  Canary is the recommended preview channel, because it ships at the fastest cadence and has the newest APIs.
+
+*  Use the Evergreen WebView2 Runtime when you use a release version of the WebView2 SDK package.
+
+For more information, see [Matching the Runtime version with the SDK version](concepts/versioning.md#matching-the-runtime-version-with-the-sdk-version).
+
+
+<!-- ====================================================================== -->
+## Minimum version of the browser or Runtime to load WebView2
+
+To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.
+
+To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
+
+
+<!-- ====================================================================== -->
+## 1.0.1150.38
+  
+Release Date: March 10, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1150.38](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1150.38)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 99.0. 1150.38 or higher.  
+
+### General
+
+#### Bug Fixes
+  
+*  Fixed a bug where WebView2 app gets stuck occasionally with UWP.

```

Comment:
@imicy / @vchapel - lets move this to the prerelease release notes please. Other than that looks good!

Reply:
(the bullet: Fixed a bug where WebView2 app gets stuck occasionally with UWP.)